tests/test_serializer_nested.py

```diff
@@ -302,3 +305,55 @@ class Meta:
             'serializer `tests.test_serializer_nested.DottedAddressSerializer`, '
             'or set `read_only=True` on dotted-source serializer fields.'
         )
+
+
+@pytest.mark.skipif('not postgres_fields')
+class TestNestedNonRelationalFieldWrite:
+    """
+    Test that raise_errors_on_nested_writes does not raise `AssertionError` when the
+    model field is not a relation.
+    """
+
+    def test_nested_serializer_create_and_update(self):
+        class NonRelationalPersonModel(models.Model):
```

Comment:
The model should probably be defined outside of the class, as the duplicate definition is generating this warning: https://travis-ci.org/encode/django-rest-framework/jobs/581419355#L323-L325

Reply:
At first I tried the following defining the model inside of the Test class:
https://github.com/encode/django-rest-framework/blob/2b512c0a04e995d11538e1197684b18385ecf4fa/tests/test_serializer_nested.py#L311-L323
But I had errors with `postgres_fields` being None.

I will try declaring the Model outside of the class. I am not sure if we should also add `@pytest.mark.skipif(not postgres_fields, reason='psycopg2 is not installed')` decorator too.

Reply:
How about:

```python
if postgres_fields:
    class NonRelationalPersonModel(models.Model): 
        """Model declaring a postgres JSONField""" 
        data = postgres_fields.JSONField() 

@pytest.mark.skipif(not postgres_fields, reason='psycopg2 is not installed')
class TestNestedNonRelationalFieldWrite: 
    ...
```

Reply:
Any thoughts why the `TOXENV=base` breaks?
https://travis-ci.org/encode/django-rest-framework/jobs/581520360#L247-L265